Sprint-3/alarmclock/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Title here</title>
+    <title id="titleTime">Title here</title>
```

Comment:
This works and is fine, but because there's ever only one `<title>` in a page, we typically don't assign it an ID, but instead use a different query selector to find it - if you remove the ID here, can you make the code still work?

Reply:
fixed it 
thanks